services/github-service-impl/src/main/java/org/kontinuity/catapult/service/github/impl/kohsuke/KohsukeGitHubWebhook.java

```diff
@@ -29,7 +29,7 @@ public String getName() {
 
 	@Override
 	public String getUrl() {
-		return delegate.getUrl().toString();
+		return delegate.getConfig().get("url");
```

Comment:
I usually do, sure.  Some `private static` thing.  Also not a big deal just some habit I've developed.